packages/instantsearch.js/src/connectors/refinement-list/__tests__/connectRefinementList-test.ts

```diff
@@ -1598,6 +1598,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/refinement-
 
       const helper = jsHelper(
         createSearchClient({
+          // @ts-ignore v5 does not have this method, but it's easier to have it here. In a future version we can replace this method and its usages with search({ type: 'facet })
```

Comment:
we have a wrapped named `searchForFacets` purely for typing purposes if that helps

Reply:
the comment is more for the /lite in v5 which doesn't have sffv anymore, and the order which is checked in InstantSearch is `client.sffv, client.initIndex?.sffv, client.search`, and so as we're in v5 here technically we should only mock `client.search`, but that's ok